### ISSUE TITLE
MacOS Setup Info

### ISSUE THREAD

**Issue by brendandebeasi** [greatscottgadgets/greatfet-tutorials#18]
Could somebody provide some guidance on setting up udev rules on MacOS?

this doesn't work on MacOS Ventura:
`sudo wget https://raw.githubusercontent.com/greatscottgadgets/greatfet/master/host/util/54-greatfet.rules -O /etc/udev/rules.d/54-greatfet.rules`
`sudo udevadm control --reload-rules\`

Thanks in advance!

**Comment by brendandebeasi** [comment]
Maybe this step isn't necessary on MacOS actually? Seems like it's working fine. If somebody can confirm I can send a PR for the docs to mention this.

**Comment by martinling** [comment]
Yeah udev is a completely Linux-specific mechanism, it doesn't exist on other operating systems. There's no need for that step on MacOS.

**Comment by brendandebeasi** [comment]
Thank you @martinling 

In addition, to build this from source on MacOS you need to run:
```
virtualenv venv
source venv/bin/activate
pip install ipython
```

If you install the latest release shell autocomplete doesn't work properly due to the newer version of Python3
